modules/wpml/wpml-config.php

```diff
@@ -480,11 +480,21 @@ protected function extract_blocks_parsing_rules() {
 							break;
 
 						case 'key':
-							$rule = $this->get_field_attribute( $child, 'name' );
+							$rule = $this->get_field_attributes( $child );
 							break;
 					}
 
-					if ( '' !== $rule ) {
+					if ( '' === $rule || ( is_array( $rule ) && empty( array_filter( $rule ) ) ) ) {
+						continue;
+					}
+
+					if ( is_array( $rule ) ) {
+						if ( isset( $parsing_rules[ $child_tag ][ $block_name ] ) ) {
+							$parsing_rules[ $child_tag ][ $block_name ] = array_merge( $parsing_rules[ $child_tag ][ $block_name ], $rule );
+						} else {
+							$parsing_rules[ $child_tag ][ $block_name ] = $rule;
+						}
+					} else {
```

Comment:
To avoid the same test 2 times:
```suggestion
					if ( is_array( $rule ) ) {
						$rule = array_filter( $rule );
						if ( empty( $rule ) ) {
							continue;
						}
						
						if ( isset( $parsing_rules[ $child_tag ][ $block_name ] ) ) {
							$parsing_rules[ $child_tag ][ $block_name ] = array_merge( $parsing_rules[ $child_tag ][ $block_name ], $rule );
						} else {
							$parsing_rules[ $child_tag ][ $block_name ] = $rule;
						}
					} elseif ( '' !== $rule ) {
```